saleor/product/models.py

```diff
@@ -157,7 +157,19 @@ class Meta:
             ('view_product',
              pgettext_lazy('Permission description', 'Can view products')),
             ('edit_product',
-             pgettext_lazy('Permission description', 'Can edit products')))
+             pgettext_lazy('Permission description', 'Can edit products')),
+            ('view_classes',
+             pgettext_lazy('Permission description',
+                           'Can view product types')),
```

Comment:
If it's going to take multiple lines anyway, please fold after the opening parenthesis/bracket/brace and indent.